CHANGELOG.md

```diff
@@ -4,43 +4,67 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
 
-## Unreleased Phase 3
+## Unreleased
 
-### Breaking Changes
+### MIGRATION notes
+
+#### RDS Phase 3
+
+This release includes updates that remove existing DynamoDB tables as part of
+release deployment process.   This release *cannot* be properly rolled back in
+production as redeploying a prior version of Cumulus will not recover the
+associated Dynamo tables.
+
+Please read the full change log for RDS Phase 3 and consult the [RDS Phase 3 update
+documentation](https://nasa.github.io/cumulus/docs/next/upgrade-notes/upgrade-rds-phase-3-release)
+
+#### API Endpoint Versioning
+
+As part of the work on CUMULUS-3072, we have added a required header for the
+granule PUT/PATCH endpoints -- to ensure that older clients/utilities do not
+unexpectedly make destructive use of those endpoints, a validation check of a
+header value against supported versions has been implemented.
+
+Moving forward, if a breaking change is made to an existing endpoint that
+requires user updates, as part of that update we will set the current version of
+the core API and require a header that confirms the client is compatible with
```

Comment:
```suggestion
the Cumulus API and require a header that confirms the client is compatible with
```

Reply:
I guess our next sprint has the work to document the PUT/PATCH changes. https://bugs.earthdata.nasa.gov/browse/CUMULUS-3076

I suspect those instructions are going to end up in the API docs. Do we want to link that here preemptively?